pwndbg/aglib/symbol.py

```diff
@@ -106,6 +106,9 @@ def resolve_addr(addr: int) -> str | None:
     Resolution is performed in the following order:
     - Global scope symbols.
     """
+    if addr < 0:
+        return None
```

Comment:
Do we need this? Just assert maybe or remove this check?

Reply:
I can remove this from this PR. Recently someone is sending here negative addresses :/

Reply:
deleted from this PR